lib/components/app/not-found.js

```diff
@@ -13,13 +14,15 @@ const StyledAlert = styled(Alert)`
  * Displays a not-found alert if some content is not found.
  */
 const NotFound = () => (
-  <StyledAlert bsStyle='warning'>
+  <StyledAlert bsStyle="warning">
     <h1>
-      <Icon type='exclamation-triangle' withSpace />
-      <FormattedMessage id='components.NotFound.header' />
+      <StyledIconWrapper spaceRight>
+        <ExclamationTriangle />
+      </StyledIconWrapper>
```

Comment:
Yeah, these are certainly not bad ideas. I think I chose this approach originally so the wrapper could be just a simple styled component, but it grew in a more complex component anyway. It would take me a while to refactor everything to this structure, though. I'll implement it as a trial run to see what you think. Check out 7695e36